src/BTVNanoCommissioning/workflows/ttbar_validation.py

```diff
@@ -123,7 +123,7 @@ def process(self, events):
         output = self.accumulator.identity()
 
         dataset = events.metadata["dataset"]
-        output["sumw"][dataset] += ak.sum(events.genWeight)
+        output["sumw"] += ak.sum(events.genWeight)
```

Comment:
concerning the beginning of this file: can we also use code from histogrammer.py for defining the axes needed for this workflow?